libsrc/_DEVELOPMENT/l/8080/l_jpix_8080.asm

```diff
@@ -0,0 +1,11 @@
+SECTION data_clib
+
+PUBLIC l_jpix_8080
+
+l_jpix_8080:
+   db $c3 ;jp $0000
+
+fake_ix:
```

Comment:
is the `fake_ix` and split of jp even needed, if the `l_setix_8080` does access `l_jpix_8080` symbol any way? ( `ld (l_jpix_8080+1),a`)

Would make sense to *me* to do `jp some_error_trap_function` with ``l_jpix_8080` label and not even add the `fake_ix` symbol at all, it looks unused.

(I'm not very familiar with z88dk, so maybe I'm missing some technical benefit of this approach)

Reply:
As for "what should the default jump address be?". I don't think there's any z88dk general purpose, low level error handling default function available, and this code should never even be called without the code to set ix/iy it being run first. l_jpix_8080 should not be used by user written programs, only internally by z88dk's clib. I'll tell you what, I'll add a ret instruction to both l_jpix_8080 and l_jpiy_8080 and have the default addresses point to them. If in the future someone modifies the clib to call jpix_8080 without first calling setix_8080 first, it'll just return invalid results back to the calling code rather than jumping to $0000 which would cause a system restart. After that adding unit tests to the clib for new functionality would have to provide any further assurances.

Reply:
Oh, and to clarify the technical benefit of this approach, the existing l_bsearch and l_qsort for the z80 were using the ix and iy registers. The 8080 and gbz80 CPUs don't have these registers :). Plus it would be much harder to do function pointers without something that performs the same function as them. The HL register pair is used by z88dk as a 16 bit scratchpad register, so using "jp (hl)" would be hard to make work as its value doesn't stay unchanged for long.